cdogs-sdl.desktop

```
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=C-Dogs SDL
+GenericName=Action/Arcade Game
+Version=1.0
+Type=Application
+Categories=Game;ActionGame;
+Keywords=Game;Arcade;Shooter;Retro;
+Comment=Classic overhead run-and-gun game
+TryExec=cdogs-sdl
+Exec=cdogs-sdl
+Terminal=false
+Icon=cdogs-sdl
```

```suggestion
Icon=io.github.cxong.cdogs-sdl
```
Also rename the file & remove the rename-* things from the manifest.